webapp/components/Chart/utils/getChartOptions.ts

```diff
@@ -0,0 +1,96 @@
+import * as NumberUtils from '@common/bignumberUtils'
+import { ChartType } from './ChartType'
+
+const commonOptions = {
+  maintainAspectRatio: false,
+  responsive: true,
+  responsiveAnimationDuration: 200,
+  tooltips: {
+    backgroundColor: 'rgba(45, 45, 45, 0.95)',
+    caretSize: 4,
+    cornerRadius: 3,
+    position: 'average',
+    xPadding: 12,
+    yPadding: 18,
+    titleAlign: 'center',
+    titleFontFamily: `'Open Sans', sans-serif`,
+    titleFontSize: 14,
+    titleFontStyle: '600',
+    titleMarginBottom: 12,
+    bodyFontFamily: `'Open Sans', sans-serif`,
+    bodyFontSize: 13,
+    bodySpacing: 6,
+    callbacks: {
+      label: (tooltipItem: Record<string, string | number>, data: any) => {
```

Comment:
Docs are weird in this, not sure about the type for `data`.
Also, `TooltipItem` -type is not exported from chart.js?